### PR TITLE
Added Whole System History API documentation to swagger and open api gen

### PR DESCRIPTION
**Pull request by mpayne2**
Signed-off-by: mpayne2 <mpayne2@us.ibm.com>

### CODE REVIEW

File: fhir-swagger-generator/src/main/java/com/ibm/fhir/openapi/generator/FHIROpenApiGenerator.java [hunk 1]
```diff
@@ -138,6 +138,7 @@
     public static final String TYPEPACKAGENAME = "com.ibm.fhir.model.type";
```
Comment (tbieste) [inline fhir-swagger-generator/src/main/java/com/ibm/fhir/openapi/generator/FHIROpenApiGenerator.java:138]: Update copyright at top of file to "2018, 2022".

File: fhir-swagger-generator/src/main/java/com/ibm/fhir/swagger/generator/FHIRSwaggerGenerator.java [hunk 2]
```diff
@@ -190,6 +190,7 @@ public static void main(String[] args) throws Exception {
```
Comment (tbieste) [inline fhir-swagger-generator/src/main/java/com/ibm/fhir/swagger/generator/FHIRSwaggerGenerator.java:190]: Update copyright at top of file to "2016, 2022".

File: fhir-swagger-generator/src/main/java/com/ibm/fhir/swagger/generator/FHIRSwaggerGenerator.java [hunk 3]
```diff
@@ -349,6 +354,75 @@ private static void generateMetadataSwagger() throws Exception, ClassNotFoundExc
...
+        // FHIR _history interaction
+        JsonObjectBuilder path = factory.createObjectBuilder();
+        generateWholeSystemHistoryPathItem(path, "Other", "Get the whole system history", true);
+        paths.add("/_history", path);
+        swagger.add("paths", paths);
+
+
+        generateDefinition(Bundle.class, definitions);
+        generateDefinition(Resource.class, definitions);
+//        generateDefinition(DomainResource.class, definitions);
```
Comment (tbieste) [inline fhir-swagger-generator/src/main/java/com/ibm/fhir/swagger/generator/FHIRSwaggerGenerator.java:385]: Is this commented out line needed?

**Review by tbieste — COMMENTED** [review]
When updating these, I think we also usually update /fhir-openapi/src/main/webapp/META-INF/openapi.json with the generated OpenAPI file as well.

**Comment by mpayne2** [comment]
I'm not sure which generated open api file I should use to copy this into /fhir-openapi/src/main/webapp/META-INF/openapi.json.  The all-openapi.json has many more differences than what is in this file.

**Review by tbieste — APPROVED** [review]
LGTM

**Review by lmsurpre — CHANGES_REQUESTED** [review]
The code looks good, but when I us the generated openapi.json in Liberty it emits a bunch of warnings like the following:
```
[2/3/22, 13:24:45:907 UTC] 00000034 OpenAPIUtils  E   CWWKO1650E: Validation of the OpenAPI document produced the following error(s): 
                                 
 - Message: More than one Operation Objects with "_history" value for "operationId" field was found. The "operationId" must be unique among all operations described in the API, Location: #/paths/~1ActivityDefinition~1_history/get/operationId
 ```
 
 please address

**Comment by mpayne2** [comment]
I can't amend previous commits with the signoff.  Moving to a new pull request.

**Comment by mpayne2** [comment]
New pull request:
https://github.com/IBM/FHIR/pull/3279

**Pull request closed without merging**